layout/css/main.css

```diff
@@ -0,0 +1,102 @@
+*{
+  padding:0;
+  margin:0;
+}
+
+body {
+  font-family: sans-serif;
+}
+
+#buttonContainer{
```

Comment:
can we do kebab-case for these to work with the style guide https://github.com/USGS-VIZLAB/vizlab/wiki/Style-guide?